support/mender-inventory-geo

```diff
@@ -0,0 +1,64 @@
+#!/bin/sh
+#
+# Example script called by Mender agent to collect inventory data for a
+# particular devce. The script needs to be located in $(datadir)/mender and its
+# name shall start with `mender-inventory-` prefix. The script shall exit with
+# non-0 status on errors. In this case the agent will discard any output the
+# script may have produced.
+#
+# The script shall output inventory data in <key>=<value> format, one entry per
+# line. Entries appearing multiple times will be joined in a list under the same
+# key.
+#
+# $ ./mender-inventory-geo
+# country=Poland
+# city=Krakow
+# latitude=50.05750
+# longitude=19.98020
+# ipv4=127.0.0.1
+#
+# The example script collects information on geo localization
+# this can be done in one line:
+# curl https://ipvigilante.com/`curl -4 ifconfig.io 2>/dev/null` 2>/dev/null | jq -r 'select(.status == "success") | "country=" + .data.country_name, "city="+.data.city_name, "latitude="+.data.latitude, "longitude="+.data.longitude, "ipv4="+.data.ipv4 | @text'
+# but in the following we assume busybox version of the utilities.
+
+# this we will use for everything, wget in busybox does not support -O-
+tmpfile=`mktemp`;
+# setup cleanup
+function geo_cleanup() {
+ rm -f "${tmpfile}";
+}
+
+trap geo_cleanup EXIT;
+trap geo_cleanup SIGHUP;
+trap geo_cleanup SIGQUIT;
+trap geo_cleanup SIGINT;
+trap geo_cleanup SIGTERM;
+trap geo_cleanup SIGKILL;
+
+# find the ip address
+wget -qO "${tmpfile}" --header "Content-Type: text/plain;" \
+     --header 'Host: ifconfig.io' --header 'User-Agent: curl/7.67.0' \
+     --header 'Accept: */*' ifconfig.io;
+
+[ -f "${tmpfile}" ] || exit 1;
+
+ip=`cat "${tmpfile}" | tail -1`;
+
+[ "${ip}" != "" ] || exit 2;
+
+# get the information, see https://www.ipvigilante.com/api-developer-docs/
+wget -qO "${tmpfile}" https://ipvigilante.com/csv/${ip} 2>/dev/null;
+
+[ -s "${tmpfile}" ] || exit 3;
+
+cat "${tmpfile}" | awk -v ip="a-ip" -v continent="a-continent" \
```

Comment:
Suggestion:
Maybe the variables you define can simply be hard-coded in the `awk` script?

Reply:
actually I wanted to make them configurable. changed.